Problem.cpp

```diff
@@ -519,29 +519,106 @@ void Problem::print_sources() {
 
 /*  ===============   Data Output  ===============  */
 
-void Problem::write_output(bool save_valid = false, bool save_solution = false) {
+void Problem::write_valid() {
```

Comment:
It bothers me a bit that the name is "write_valid" instead of "write_validation", because it sounds like it is "writing valid data" and not "writing _validation_ data"... Do you mind changing it, please? In the name of the file also...